src/app/workflow/versions/versions.component.html

```diff
@@ -15,170 +15,164 @@
-              <label placement="top" matTooltip="Version of the workflow.">
-                Version
-              </label>
-            </span>
-            <fa-icon class="pull-right" [icon]="getIconClass('reference')" (click)="clickSortColumn('reference')"></fa-icon>
-          </th>
+    <ng-container matColumnDef="last_modified">
+      <th scope="col" mat-header-cell *matHeaderCellDef mat-sort-header disableClear matTooltip="Date of last update to Git reference">
+        Date Modified
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <span matTooltip="{{ version.last_modified | date: 'medium' }}">{{ version.last_modified | date: 'mediumDate' }}</span>
+      </td>
+    </ng-container>
 
-          <th *ngIf="workflow?.mode === WorkflowType.ModeEnum.HOSTED">
-            <fa-icon class="pull-right" [icon]="getIconClass('author')" (click)="clickSortColumn('author')"></fa-icon>
-            <label placement="top" matTooltip="The editing author as collected by Dockstore">Author</label>
-          </th>
-          <th>
-            <fa-icon class="pull-right" [icon]="getIconClass('last_modified')" (click)="clickSortColumn('last_modified')"></fa-icon>
-            <label placement="top" matTooltip="Date of last update to Git reference">Date Modified</label>
-          </th>
+    <ng-container matColumnDef="valid">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="A version is valid if it has at least one valid descriptor file."
+      >
+        Valid
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <mat-icon *ngIf="version.valid">check</mat-icon>
+      </td>
+    </ng-container>
 
-          <th>
-            <fa-icon class="pull-right" [icon]="getIconClass('valid')" (click)="clickSortColumn('valid')"></fa-icon>
-            <label placement="top" matTooltip="A version is valid if it has at least one valid descriptor file">
-              Valid
-            </label>
-          </th>
+    <ng-container matColumnDef="hidden">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="A hidden version is only visible here and not publicly."
+      >
+        Hidden
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <mat-icon *ngIf="version.hidden">check</mat-icon>
+      </td>
+    </ng-container>
 
-          <th *ngIf="!publicPage">
-            <fa-icon class="pull-right" [icon]="getIconClass('hidden')" (click)="clickSortColumn('hidden')"></fa-icon>
-            <label placement="top" matTooltip="A hidden version is only visible here and not publicly.">
-              Hidden
-            </label>
-          </th>
+    <ng-container matColumnDef="verified">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        matTooltip="A version is verified if it has been verified to work by a person/group."
```

Comment:
There's a `href="{{ Dockstore.DOCUMENTATION_URL }}/faq.html#what-is-a-default-version-of-a-tool-or-workflow"` above, can we get that here too but linked to https://docs.dockstore.org/en/develop/advanced-topics/verification.html

Reply:
Tooltips with links aren't a thing

Reply:
Yeah, I may not be commenting on the right line. 
But the Git Reference column uses a bubble, the verified column seems to link the checkmark. 
Let's make it consistent by using the bubble in both places and removing the link

Reply:
Splitting this out into separate ticket because the Git Reference column's bubble has disappeared because setting default version is now done in the dropdown and not on the column anymore.  Not sure if we want the lone verified bubble anymore

Reply:
Ok, not splitting it out. It does look weird though